packages/frontend/src/components/ExploreSideBar.tsx

```diff
@@ -234,11 +234,12 @@ export const ExploreSideBar = () => {
         state: { tableName },
         actions: { reset },
     } = useExplorer();
+    const { projectUuid } = useParams<{ projectUuid: string }>();
```

Comment:
Components are pulling project directly from the path params. Not sure if you prefer this in a hook @ZeRego ?